config/initializers/sentry.rb

```diff
@@ -0,0 +1,14 @@
+Sentry.init do |config|
+  return unless ENV.has_key?('SENTRY_DSN')
+  config.dsn = ENV.fetch('SENTRY_DSN')
+  config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+
+  # Set tracesSampleRate to 1.0 to capture 100%
+  # of transactions for performance monitoring.
+  # We recommend adjusting this value in production
+  # config.traces_sample_rate = 0.5
```

Comment:
I'm assuming that if we need to tune this in production later, we'd do so using ENV vars - but that the volume of this application may allow us to run defaults?

Reply:
I don't think we even have access to tracing in our Sentry so this boilerplate comment is probably not super helpful.. but since it was boilerplate I left it.